libs/platform/src/lib/components/menu/menu-trigger.directive.ts

```diff
@@ -242,6 +241,9 @@ export class MenuTriggerDirective implements OnDestroy, AfterContentInit {
         let positions: ConnectedPosition[] = [];
         const offsetYPosition = 0;
         const offsetXPosition = 0;
+        const subMenuXPadding = 4; // horizontal padding of 0.25rem(4px) is needed for sub-menu
```

Comment:
Could you please add this as a input variable with default value

Reply:
fiori design has fix value of these padding. user input will not be necessary.